### DISCUSSION TITLE
How to set base URLs on IOS when using relative URLs in ReactJS

### DISCUSSION THREAD

**Discussion by henryhs97** [ionic-team/capacitor#5077] [Q&A]
Hi! I'm trying to build a capacitor iOS app based on a ReactJS web app I have. Everything so far is good, except that I cannot do any web requests. My suspicion is that it's because I'm using relative URLs in my web app. For example, `fetch('/api/routines')`, instead of also using the base URL (i.e. `fetch('https://mywebsite.com/api/routines')`). The most straightforward fix I can think of is to update all the requests in my app to also include the base URL, but this will be quite some work so I was wondering if there was a way to set the capacitor config so that the base URL can be defined? Any other ideas are very much welcome. Thanks in advance!

**Comment by cdharbinger** [comment]
Hello @henryhs97 

Did you find any solution for this?

**Reply by zdu-strong** [reply]
You might consider using axios.
It can meet your needs.

**Comment by zdu-strong** [comment]
https://axios-http.com

it is used like this:
axios.get('/api/routines')
